lib/Controller/Display.php

```diff
@@ -2275,6 +2275,10 @@ public function moveCms(Request $request, Response $response, $id)
                 throw new InvalidArgumentException(__('Provided CMS URL is invalid'), 'newCmsUrl');
             }
 
+            if (!v::stringType()->length(1, 1000)->validate($newCmsAddress)) {
```

Comment:
Doesn't a length from 1 to 1000 mean that we must enter a new CMS URL?

Reply:
Well, yes, couple of lines above that we check if it's empty anyway ie it cannot be empty (or invalid or now over 1000 characters).

Is that not correct?

Reply:
Understood, makes sense if its always required! :+1: